JsonSchema/JsonSchemaSerializerContext.cs

```diff
@@ -73,11 +75,9 @@ namespace Json.Schema;
 [JsonSerializable(typeof(VocabularyKeyword))]
 [JsonSerializable(typeof(WriteOnlyKeyword))]
 [JsonSerializable(typeof(decimal))]
+[JsonSerializable(typeof(SchemaOrPropertyList))]
 [JsonSerializable(typeof(Dictionary<string, bool>))]
-[JsonSerializable(typeof(Dictionary<string, JsonSchema>))]
-[JsonSerializable(typeof(Dictionary<string, List<string>>))]
 [JsonSerializable(typeof(Dictionary<string, PropertyDependency>))]
-[JsonSerializable(typeof(Dictionary<string, SchemaOrPropertyList>))]
 [JsonSerializable(typeof(Dictionary<string, string[]>))]
 [JsonSerializable(typeof(IReadOnlyCollection<JsonNode?>))]
```

Comment:
I didn't finish this last night, but we should remove all the collection types off here just to be safe.